graphql_compiler/compiler/sqlalchemy_extensions.py

```diff
@@ -40,3 +40,26 @@ def not_contains_operator(collection, element):
                              .format(sqlalchemy.sql.schema.Column, type(element)))
 
     return element.notin_(collection)
+
+
+def print_sqlalchemy_query_string(query, dialect):
+    """Return a string form of the parameterized query.
+
+    Args:
+        query: sqlalchemy.sql.selectable.Select
+        dialect: sqlalchemy.engine.interfaces.Dialect
+
+    Returns:
+        string that can be ran using sqlalchemy.sql.text(result)
+    """
+    class LiteralCompiler(sqlalchemy.dialects.mssql.base.MSSQLCompiler):
```

Comment:
Can you either make sure this works for dialects other than mssql, raise an assertion if this is not a mssql dialect,  or change the docstring of this function?

Reply:
Made it dialect-agnostic